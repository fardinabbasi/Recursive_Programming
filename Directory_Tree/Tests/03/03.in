4 3
folder1 0
folder2 1
file1 2
file2 1