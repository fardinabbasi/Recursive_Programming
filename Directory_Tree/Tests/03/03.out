|__folder1
   |__folder2
   |  |__file1
   |__file2